tutorials/notebooks/astropy-coordinates/3-Coordinates-Velocities.ipynb

```diff
@@ -0,0 +1,529 @@
+    "    pm_dec=np.linspace(-5, 5, 5)*u.mas/u.yr)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "However, for some of the examples below we will continue to use scalar values for brevity. \n",
+    "\n",
+    "You can also specify radial velocity data with the `radial_velocity` argument:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "velocity_coord = SkyCoord(\n",
+    "    ra=10*u.deg, \n",
+    "    dec=20*u.deg,\n",
+    "    pm_ra_cosdec=1*u.mas/u.yr,\n",
+    "    pm_dec=2*u.mas/u.yr,\n",
+    "    radial_velocity=100*u.km/u.s)\n",
+    "velocity_coord"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The component data can then be accessed using the same names used to pass in the velocity components:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "velocity_coord.pm_ra_cosdec"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "velocity_coord.radial_velocity"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "A `SkyCoord` object with velocity data can be transformed to other frames just like the position-only coordinate objects we used in the previous tutorials:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "velocity_coord_gal = velocity_coord.transform_to(Galactic())\n",
+    "velocity_coord_gal"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Note that, like the position components, which change from `ra`,`dec` to `l`,`b`, the proper motion component names have changed to reflect naming conventions for the component names in a given frame: `pm_ra_cosdec` and `pm_dec` have become `pm_l_cosb` and `pm_b`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "velocity_coord_gal.pm_l_cosb"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "velocity_coord_gal.pm_b"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "An important caveat to note when transforming a `SkyCoord` object with velocity data is that some reference frames require knowing the distances, or the full velocity vectors (i.e. proper motion components and radial velocity) in order to transform the velocities correctly. For example, a `SkyCoord` with only sky position and proper motion data cannot be transformed to a frame with a positional or velocity offset, such as the `Galactocentric` frame ([docs](https://docs.astropy.org/en/stable/coordinates/galactocentric.html))"
```

Comment:
Perhaps it would be interesting to introduce this caveat earlier, in tutorial 2? It also happens when there are no velocities:

```
In [5]: test_coord = SkyCoord(
   ...:     ra=10*u.deg,
   ...:     dec=20*u.deg,
   ...: )
   ...: 
   ...: # This cell will raise an exception - this is expected!
   ...: test_coord.transform_to(coord.Galactocentric())
---------------------------------------------------------------------------
ConvertError                              Traceback (most recent call last)
<ipython-input-5-f0d341cd68ec> in <module>
...
~/Personal/Astropy/astropy/astropy/coordinates/builtin_frames/galactocentric.py in _check_coord_repr_diff_types(c)
    561 def _check_coord_repr_diff_types(c):
    562     if isinstance(c.data, r.UnitSphericalRepresentation):
--> 563         raise ConvertError("Transforming to/from a Galactocentric frame "
    564                            "requires a 3D coordinate, e.g. (angle, angle, "
    565                            "distance) or (x, y, z).")

ConvertError: Transforming to/from a Galactocentric frame requires a 3D coordinate, e.g. (angle, angle, distance) or (x, y, z).
```
```

Reply:
👍